modules/n1ql/pages/n1ql-language-reference/stringfun.adoc

```diff
@@ -1042,3 +1042,45 @@ SELECT UPPER("N1QL is awesome") as n1ql;
 }
 ----
 ====
+
+[#fn-str-upper]
+== URLENCODE(plain_string)
+
+=== Description
+Returns the input string encoded for use in a URL
+
+=== Arguments
+in_str:: A string, or any valid xref:n1ql-language-reference/index.adoc[expression] which evaluates to a string, that is the string to convert to upper case.
```

Comment:
Needs to be consistent with syntax given in the function title. Also fix description:

```suggestion
plain_string:: A string, or any valid xref:n1ql-language-reference/index.adoc[expression] which evaluates to a string, that is the string to URL-encode.
```